Makefile

```diff
@@ -8,12 +8,12 @@ XDR_REF?=main
 CORE_REPO?=https://github.com/stellar/stellar-core.git
 CORE_REF?=master
 CORE_CONFIGURE_FLAGS?=--disable-tests
-SOROBAN_RPC_REF?=main
-HORIZON_REF?=$(shell ./scripts/soroban_repo_to_horizon_repo.sh $(SOROBAN_RPC_REF))
+STELLAR_RPC_REF?=main
+HORIZON_REF?=$(shell ./scripts/soroban_repo_to_horizon_repo.sh $(STELLAR_RPC_REF))
```

Comment:
does this script need to be renamed?

Reply:
This doesn't need addressing in this PR imo.

Reply:
Good point, I did.